src/main/java/com/pubnub/api/managers/DelayedReconnectionManager.java

```diff
@@ -5,17 +5,21 @@
 import com.pubnub.api.enums.PNReconnectionPolicy;
 import lombok.extern.slf4j.Slf4j;
 
+import java.text.DecimalFormat;
+import java.util.Random;
 import java.util.Timer;
 import java.util.TimerTask;
 
 @Slf4j
 public class DelayedReconnectionManager {
     private static final int DELAY_SECONDS = 3;
+    private static final int MAX_RANDOM = 3;
```

Comment:
we're changing this to 1 second, right?

Reply:
Good catch.
Yes, for new approach we decided to go with random 0.000 to 0,999. Initially for TataSky I thought I will go with 3. but after second thoughts I will change it to 1.